### PR TITLE
Unify keyboard movement with joystick movement

### LINKED ISSUE #360: [FEATURE] Fix keyboard movement
## 🚀 Feature

Make the keyboard movement map to joystick values so to the backend it's all the same and diagonal movement is fixed.

### PR DESCRIPTION
**Pull request by jrchatruc**
Closes #360

### CODE REVIEW

File: client/Assets/Scripts/PlayerControls.cs [hunk 1]
```diff
@@ -44,57 +44,29 @@ public void SendJoystickValues(float x, float y)
...
 
-            ClientPrediction.PlayerInput playerInput = new ClientPrediction.PlayerInput
-            {
-                joystick_x_value = 0f,
-                joystick_y_value = -1f,
-                timestamp = timestamp,
-            };
-            SocketConnectionManager.Instance.clientPrediction.putPlayerInput(playerInput);
+        if (direction.x !=0 || direction.y != 0) {
+            var valuesToSend = new RelativePosition { X = direction.x, Y = direction.y };
```
Comment (tvillegas98) [inline client/Assets/Scripts/PlayerControls.cs:66]: Can we use SendJoystickValues here, can't we?

Comment (jrchatruc) [reply client/Assets/Scripts/PlayerControls.cs]: Yeah you're right, I changed it here [9a77c6a](https://github.com/lambdaclass/curse_of_myrra/pull/397/commits/9a77c6ae4338167eb7d224b7814f40d33b6e6474)

**Review by tvillegas98 — APPROVED** [review]

**Comment by tvillegas98** [comment]
LGTM!

**Review by BertovDev — APPROVED** [review]
LGTM

**Pull request merged**